packages/app/src/components/PageEditor/EmojiPicker.tsx

```diff
@@ -13,9 +16,9 @@ const EmojiPicker: FC<Props> = (props: Props) => {
   const { close, emojiSearchText, editor } = props;
 
   const emojiPickerContainer = useRef<HTMLDivElement>(null);
-
+  const [isDarkMode, setIsDarkMode] = useState(isDarkModeByUtil());
   useEffect(() => {
-
+    setIsDarkMode(isDarkModeByUtil());
```

Comment:
Is it necessary to use useState for isDarkMode?
if it's not, please remove it.

Reply:
Resolved in https://github.com/weseek/growi/pull/5606/commits/b384d1ddcbb3c1743159437d221ed696d8c3d06a